src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java

```diff
@@ -29,10 +32,15 @@
     @Given("^username \"([^\"]*)\" and password \"([^\"]*)\"$")
     public void usernameAndPassword(final String username, final String password) {
         Dashboard dashboard;
-        dashboard = SignInPage.newCredentials(Environment.getInstance().getProperties()
-                .getProperty(username),
-                Environment.getInstance().getProperties()
-                        .getProperty(password));
+        try {
+            dashboard = SignInPage.newCredentials(Environment.getInstance().getProperties()
+                    .getProperty(username), Environment.getInstance().getProperties()
+                    .getProperty(password));
+        } catch (NullPointerException ex) {
+            LOGGER.error("The parameters of username or password are missing in the config.properties file");
+            throw new RuntimeException();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)